a8c-files.php

```diff
@@ -86,13 +86,15 @@ public function __construct() {
 		// ensure the correct upload URL is used even after switch_to_blog is called
 		add_filter( 'option_upload_url_path', array( $this, 'upload_url_path' ) );
 
-		// Conditionally schedule the attachment filesize metaata update job
+		// Conditionally schedule the attachment filesize metadata update job
 		if ( defined( 'VIP_FILESYSTEM_SCHEDULE_FILESIZE_UPDATE' ) && true === VIP_FILESYSTEM_SCHEDULE_FILESIZE_UPDATE ) {
 			// add new cron schedule for filesize update
 			add_filter( 'cron_schedules', array( $this, 'filter_cron_schedules' ), 10, 1 ); // phpcs:ignore WordPress.WP.CronInterval.CronSchedulesInterval
 
-			// Schedule meta update job
-			$this->schedule_update_job();
+			if ( defined( 'WP_CLI' ) && WP_CLI ) {
```

Comment:
If we ever move away from `WP_CLI`-based cron execution this will break.

Slightly better would be to use `is_admin()` (we could qualify to further to skip admin ajax as well if we want).

Reply:
@mjangda if we ever decide to do that we'll have to audit everything regardless, I think, but fair point, I added a `wp_doing_cron` check for future proofing.